subprojects/docs/src/docs/userguide/javaSoftware.adoc

```diff
@@ -1,523 +0,0 @@
-// Copyright 2017 the original author or authors.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-[[java_software]]
```

Comment:
The page you’re removing (java software) gets about 1000 hits/month. Seems like we should add a redirect here or keep the page and say ”don't do it!" like we do with the Jetty plugin. 

We should still remove the link from the nav, though.

Reply:
In the beginning I only said "don't do it". I removed the page based on the comments from @jjohannes and @oehme (see above). I think people go to the page because they are actually trying to look for either the Java plugin or the Java library plugin of the current model. The page I used a lot of "Java Library" references. IMHO I don't even want people to find this page anymore. It leads plugin developers and users down the wrong path.

> We should still remove the link from the nav, though.

It's already removed.

Reply:
I added an issue to set up a redirect: https://github.com/gradle/dotorg-docs/issues/190